### ISSUE TITLE
Adding .Net Support to ALT:V to load DLL Files

### ISSUE THREAD

**Issue by djdascream** [parkervcp/eggs#1148]
# If you are seeing any of the following go to Discord and port your error in a support channel.
  * `A fatal error was encountered while starting this server.`
  * `No server egg configuration could be located; aborting startup.`

Please fill out the information bellow and remove from the line up  
If you just submit a bug with no info I will close out your bug.
---------------

Panel Version: 1.4.1
Daemon Version: 1.4.2
Service: ALTV
Modified: no

Expected Behavior:
The Server Loads CS Ressources (dll Files)

Actual Behavior:
The Server cannot Load the Files. Error in Log:

It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '5.0.0' was not found.
  - The following frameworks were found:
      3.1.8 at [/usr/share/dotnet/shared/Microsoft.NETCore.App]

You can resolve the problem by installing the specified framework and/or SDK.

The specified framework can be found at:
  - https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=5.0.0&arch=x64&rid=debian.10-x64
  
  installo Informations: https://docs.microsoft.com/de-de/dotnet/core/install/linux-debian

Steps to Reproduce: (Step by step what to do to cause the issue)
Install an AltV Server, add the libcsharp-module and Start the Server

**Comment by Software-Noob** [comment]
The alt:v documentation recommends 3.1, are you sure that you need 5.0 instead of using 3.1 modules? You could edit the docker image to use 5.0, but this would most likely be a breaking change for other users.

**Comment by DatMayo** [comment]
@Software-Noob yes its necessary, due the fact that a lot of new thinks will implement .net 5 methods. I allready created a PR (unfortunately first in my own repo). Now its open on: https://github.com/parkervcp/images/pull/89
